### PR TITLE
Flux Monitor reorgs redux

### PR DESCRIPTION
**Pull request by spooktheducks**

### CODE REVIEW

**Comment by se3000** [comment]
Bors try

**Comment by se3000** [comment]
Bors merge

File: evm-contracts/src/v0.6/dev/FluxAggregator.sol [hunk 1]
```diff
@@ -498,7 +498,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
   function latestSubmission(address _oracle)
     external
     view
-    returns (int256, uint256)
+    returns (int256, uint32)
```
Comment (se3000) [inline evm-contracts/src/v0.6/dev/FluxAggregator.sol:501]: The AggregatorInterface treats this type as a `uint256`. Is there any benefit to not keeping that consistent in the external API?

File: evm-contracts/src/v0.6/dev/FluxAggregator.sol [hunk 2]
```diff
@@ -592,12 +592,34 @@ contract FluxAggregator is AggregatorInterface, Owned {
     updateAvailableFunds();
   }
 
+  function asdf(address _oracle, uint32 _queriedRoundId) private view returns (bool _eligible, uint128 _state) {
```
Comment (se3000) [inline evm-contracts/src/v0.6/dev/FluxAggregator.sol:595]: Is this still needed? If so, can we give it a better name?

Comment (spooktheducks) [reply evm-contracts/src/v0.6/dev/FluxAggregator.sol:595]: Whoops, good catch.

File: core/internal/gethwrappers/generation/abigen.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (se3000) [inline core/internal/gethwrappers/generation/abigen.sh:1]: I don't think this file is supposed to be checked in anymore.

File: core/internal/gethwrappers/generation/common.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (se3000) [inline core/internal/gethwrappers/generation/common.sh:1]: I believe we got rid of this one too.

File: core/internal/gethwrappers/generation/generate.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (se3000) [inline core/internal/gethwrappers/generation/generate.sh:1]: ✂️

File: core/internal/gethwrappers/generation/generated-wrapper-dependency-versions-do-not-edit.txt [hunk 6]
```diff
@@ -1,5 +1,6 @@
 GETH_VERSION: 1.9.12
-flux_aggregator_wrapper: ../../../evm-contracts/abi/v0.6/FluxAggregator.json bf09af4a10707e1fc1c00c533998fdc5421ae80c3e5c9d35cefc3ceeb90f7720
+flux_aggregator_wrapper: ../../../evm-contracts/abi/v0.6/FluxAggregator.json 8baca2b71a5d622ca91d1b0269d04d41387a179e0184f9930f3daa931c42ea45
+link_token_interface: ./generation/../../../../evm-test-helpers/src/LinkToken.json a16bfa4690d25769959dd59034f03d101a46ecbf7e774a57ca6874f8352c8568
```
Comment (se3000) [inline core/internal/gethwrappers/generation/generated-wrapper-dependency-versions-do-not-edit.txt:3]: link_token_interface is no longer stored in this file.

File: core/internal/gethwrappers/generation/record_versions.sh [hunk 7]
```diff
@@ -0,0 +1 @@
+
```
Comment (se3000) [inline core/internal/gethwrappers/generation/record_versions.sh:1]: ✂️

File: core/store/orm/orm.go [hunk 8]
```diff
@@ -1182,6 +1166,39 @@ func (orm *ORM) FindLogConsumer(lc *models.LogConsumption) (models.JobSpec, erro
 	return orm.FindJob(lc.JobID)
 }
 
+func (orm *ORM) FindFluxMonitorRoundStats(aggregator common.Address, roundID uint32) (models.FluxMonitorRoundStats, error) {
```
[the remark this answers is not in the document]
Comment (se3000) [reply core/store/orm/orm.go:1169]: https://www.pivotaltracker.com/story/show/173168317

File: evm-contracts/src/v0.6/dev/FluxAggregator.sol [hunk 9]
```diff
@@ -603,7 +603,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
    * only to be callable by oracles. Not for use by contracts to read state.
    * @param _oracle the address to look up information for.
    */
-  function oracleRoundState(address _oracle)
+  function oracleRoundState(address _oracle, uint32 _queriedRoundId)
```
[the remark this answers is not in the document]
Comment (se3000) [reply evm-contracts/src/v0.6/dev/FluxAggregator.sol:606]: https://www.pivotaltracker.com/story/show/173168317

File: core/store/orm/orm.go (hunk 8 above)
[the remark this answers is not in the document]
Comment (se3000) [reply core/store/orm/orm.go:1169]: https://www.pivotaltracker.com/story/show/173168326

**Pull request merged**